### ISSUE TITLE
Db connectivity 

### ISSUE THREAD

**Issue by AnimeshGaur** [coco2cool/NetBanking#1]

**Comment by coco2cool** [comment]
I'm sorry. How should there be a problem?

**Comment by evachelam1** [comment]
mysql_connect("localhost","root","");

simply edit here as below...... This is a beautiful script.....

mysql_connect("localhost","mysqluser here","mysql_password");


Thanks to coder...deeply love this

**Comment by AnimeshGaur** [comment]
is there any problem

**Comment by evachelam1** [comment]
yes the script is not working. The admin won't log in and the employee..so nothing is functioning...
Maybe upload the working script

**Comment by coco2cool** [comment]
Hi @evachelam1  and @ani8222 ,

Sorry, I wasn't monitoring this project, as I got involved in other technologies, and have mostly moved away from web development.
